bus/src/main/java/org/hawkular/metrics/component/publish/AvailDataMessage.java

```diff
@@ -104,6 +104,8 @@ public String toString() {
         @JsonInclude
         private String tenantId;
         @JsonInclude
+        private String type;
```

Comment:
It is similar as MetricDataMessage.SingleMetric.type field used to build the dataId used internally in alerting.
Probably it is not necessary as it can be used as a constant, but then if the datatype changes in metric (i.e. new availability types in the future) constant needs to be in sync and updated on alerting too.
This is a redundant field that helps to unify that flow.